data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1580,5 +1567,22 @@
     "qualities": [ { "id": "CONTAIN", "level": 1 } ],
     "tools": [ [ [ "syringe", -1 ] ], [ [ "surface_heat", 1, "LIST" ] ] ],
     "components": [ [ [ "hi_q_distillate", 10 ] ], [ [ "element", 1 ] ], [ [ "plastic_chunk", 1 ] ], [ [ "glass_shard", 1 ] ] ]
+  },
+  {
+    "result": "conc_venom",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_OTHER",
+    "skill_used": "chemistry",
+    "difficulty": 7,
+    "skills_required": [ [ "firstaid", 6 ] ],
+    "time": "60 m",
+    "batch_time_factors": [ 50, 3 ],
+    "autolearn": false,
+    "qualities": [ { "id": "CHEM", "level": 3 }, { "id": "SEPARATE", "level": 1 }, { "id": "CONCENTRATE", "level": 1 } ],
+    "tools": [ [ [ "syringe", -1 ] ], [ [ "surface_heat", 30, "LIST" ] ] ],
+    "components": [ [ [ "venom_wasp", 2 ], [ "venom_bee", 2 ] ] ],
```

Comment:
I think this recipe should probably require one venom gland (of either type), rather than two (with weight/volume of concentrated venom halved and amount of charges used in recipes doubled). So that the player doesn't have to just leave a single venom gland to rot, if they're left with it after butchering.

Reply:
Done.